source/fe/fe_q_hierarchical.cc

```diff
@@ -546,6 +749,9 @@ void FE_Q_Hierarchical<dim>::initialize_unit_support_points ()
   // Lagrange functions used in
   // FE_Q), so there's not much we
   // can do here.
+
+  // TODO shouldn't we just at least make support points unique,
+  // even though the delta property is not satisfied for this FE?
```

Comment:
Yea, I think it's wrong of this element to declare that it has support points to begin with. See issue #474